README.md

```diff
@@ -201,14 +212,190 @@ output = model.generate(
 You can also deploy the server to a cloud provider like Jina Cloud or AWS.
 To do so, you can use `deploy` command:
 
-- Jina Cloud
+### Jina Cloud
+
+using predefined executor
```

Comment:
I think `adaptor` finetuned model is an advanced use case, it is not reasonable to mention it in quickstart.